.docker/requirements.txt

```diff
@@ -1,13 +1,15 @@
-pyspelling == 2.8.2
-coverage-threshold == 0.4.4
-ptvsd == 4.3.2
-pytest-asyncio == 0.21.0
-flake8 == 6.0.0
-black == 23.3.0
-dataclasses-json == 0.5.7
-pytest-xdist == 3.3.1
-databricks-cli == 0.17.7
-mypy == 1.3.0
-build == 0.10.0
-python-dateutil == 2.8.2
+# This is a pip 'requirements.txt' file
+# See https://pip.pypa.io/en/stable/reference/requirements-file-format/
 
+#
+# PYTHON TOOLS
+#
+black
+build
+coverage-threshold
+flake8
+mypy
+ptvsd
+pyspelling
+pytest-asyncio
+pytest-xdist
```

Comment:
What are your thoughts on pinning vs not pinning the versions?
I don't have strong opinions regarding dev/CI/CD tools. But regarding production code, I prefer pinned versions to avoid sudden unexpected behaviour.

Reply:
Good point. I will pin the versions